tests/ingestion.py

```diff
@@ -0,0 +1,120 @@
+import requests
+import json
+import pystac
+from pystac import STACValidationError
+import os
+import boto3
+
+
+class StacIngestion:
+
+    """Class representing various test operations"""
+
+    def __init__(self):
+        self.ingestor_url = os.getenv("INGESTOR_URL")
+        self.stac_url = "https://stac.test.maap-project.org"
```

Comment:
if we're already using an environment file, do we want to move these as well?

Reply:
that's always going to be the same though. 

But we can easily move it as well to the workflow `env`.

Reply:
I planned on moving it to stac.dit for consistency